pkg/envoy/envoy.go

```diff
@@ -150,18 +151,35 @@ func StartEnvoy(adminPort uint32, stateDir, logPath string, baseID uint64) *Envo
 	// case no one reader reads it.
 	started := make(chan bool, 1)
 	go func() {
-		logger := &lumberjack.Logger{
-			Filename:   logPath,
-			MaxSize:    100, // megabytes
-			MaxBackups: 3,
-			MaxAge:     28,   //days
-			Compress:   true, // disabled by default
+		var logger *lumberjack.Logger
+		var logFormat string
+		if logPath != "" {
```

Comment:
I don't this should be one or the other.
I think we should tee the logs from Envoy, and at least always write into the Cilium logs.